01-import_and_maxfilter.py

```diff
@@ -273,8 +273,11 @@ def run_maxwell_filter(subject, session=None):
         os.makedirs(os.path.dirname(raw_fname_out), exist_ok=True)
 
         raw_out.save(raw_fname_out, overwrite=True)
-        if config.plot:
-            raw_out.plot(n_channels=50, butterfly=True)
+        if config.interactive:
+            (raw_out
+             .copy()
+             .pick_types(meg=True)
```

Comment:
why only meg?

Reply:
honestly I would say 

`raw_out.plot(n_channels=50, butterfly=True)`

is a wiser default. You see very quickly issues when using butterfly mode.

I am now wondering if making a copy of raw here is really a wise thing.
Can it potentially double the memory usage of the script?

Reply:
I will revert this to use butterfly mode. I think users can switch to the other view (what is it called?) by pressing `b` anyway, right? :)